### ISSUE TITLE
[Inline message] Improve accessibility of title in the inline message component

### ISSUE THREAD

**Issue by anna-lach** [sl-design-system/components#2847]
### Provide a general summary of the issue here

Title in the inline message should be a heading, so we need to improve our examples in storybook (and website) and use heading there for the title.

Use the callout component for inspiration on which heading to use (presumably h2)

check that no styling breaks

### 👤 Your name

Anna Sobczak

### 🧢 Your Product/Team

SLDS
